behat.yml.dist

```diff
@@ -131,6 +131,22 @@ default:
             filters:
                 tags: "@inventory && ~@javascript"
 
+        importexport:
```

Comment:
`import_export` looks better as your features are in directory with the same name.